flask_security/forms.py

```diff
@@ -203,8 +204,9 @@ def validate(self):
 class LoginForm(Form, NextFormMixin):
     """The default login form"""
 
-    email = StringField(get_form_field_label('email'))
```

Comment:
Can you have a look at code on line 223 if we still need it when using validators?

```python
     if self.email.data.strip() == '':
             self.email.errors.append(get_message('EMAIL_NOT_PROVIDED')[0])
             return False
 
         if self.password.data.strip() == '':
             self.password.errors.append(get_message('PASSWORD_NOT_PROVIDED')[0])
             return False
```